otx/algorithms/classification/tasks/train.py

```diff
@@ -108,7 +108,7 @@ def train(
         if train_parameters is not None:
             update_progress_callback = train_parameters.update_progress  # type: ignore
         self._time_monitor = TrainingProgressCallback(update_progress_callback)
-        self._learning_curves = defaultdict(OTXLoggerHook.Curve)
+        self._learning_curves = defaultdict(OTXLoggerHook.Curve)  # type: ignore
```

Comment:
It's been mentioned in another PR, but what does this mean?

Reply:
Will be fixed in #1473

Reply:
Fixed.